modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -325,4 +325,115 @@ spec:
 [source,terminal]
 ----
 $ oc apply -f my-performance-profile.yaml
-----
\ No newline at end of file
+----
+
+[id="node-tuning-operator-pod-power-saving-config_{context}"]
+== Optional: Power saving configurations for pods
+
+You can enable power savings for low priority workloads that are co-located with high priority workloads on an SNO node without impacting the latency or throughput of the high priority workloads. Power saving is possible without modifications to the workloads themselves.
```

Comment:
I'd suggest this:
You can enable power savings for a node that has low priority workloads that are co-located with high priority workloads without impacting the latency or throughput of the high priority workloads. Power saving is possible without modifications to the workloads themselves.

Reply:
Hmm... I think we also need to add a disclaimer here about this relying on the capabilities of the processor. Because we know that older Intel processors do not support this. Maybe something like:
You can enable power savings for a node that has low priority workloads that are co-located with high priority workloads without impacting the latency or throughput of the high priority workloads (Note: this depends on the capabilities of the processor). Power saving is possible...

Reply:
Also it should mention that only works for intel new cpus, right?

Reply:
@marioferh Added to the note with the CPU limitations:
> The feature is supported on Intel Ice Lake and later generations of Intel CPUs. The capabilities of the processor might impact the latency and throughput of the high priority workloads.